README.md

```diff
@@ -31,18 +32,28 @@ A Github organization must be created first. The organization must have the foll
 
 
 ## 2.2. Set up Github Repositories
-This application assumes 4 repositories will be created under the above organization.
+This application 4 separate repositories, one for each phase in the bug reporting process.
```

Comment:
The application has*

Reply:
I don't think it's one repo for each phase as the settings repo is used by all phases.

Reply:
Ah right, and the pe repo is used twice as well. Got to add that part in